autogen/main/README.md

```diff
@@ -213,6 +213,7 @@ The node_pools variable takes the following parameters:
 | local_ssd_count | The amount of local SSD disks that will be attached to each cluster node and may be used as a `hostpath` volume or a `local` PersistentVolume.  | 0 | Optional |
 {% if beta_cluster %}
```

Comment:
Does this need to be restricted to `beta` provider?  It appears to be available in the GA provider.

Reply:
Sure, it's GA now !